build/e2e/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG GO_VERSION=1.17
```

Comment:
We should change this to a external variable from GH since it is used in multiple places.

Also we should use a default

Reply:
i am with you, this is still early q&d, I will clean this up. same for versions of products (ZITADEL, COCKROACHDB, etc)

Reply:
Yeah my comment is more intended for further tracking 😁